youtube_dl/extractor/acast.py

```diff
@@ -48,7 +48,7 @@ class ACastIE(ACastBaseIE):
     _VALID_URL = r'''(?x)
                     https?://
                         (?:
-                            (?:(?:embed|www)\.)?acast\.com/|
+                            (?:(?:embed|www|shows)\.)?acast\.com/|
                             play\.acast\.com/s/
                         )
                         (?P<channel>[^/]+)/(?P<id>[^/#?]+)
```

Comment:
Maybe this will match the URL with `$`:
```suggestion
                            (?:(?:(?P<embed>embed)|www|shows)\.)?acast\.com|
                            play\.acast\.com/s
                        )/(?(embed)(?:$/)?)
                        (?P<channel>[^/]+)/(?P<id>[^/#?]+)
```
If so, add that URL to the tests.